deploy/examples/cluster.yaml

```diff
@@ -281,6 +281,31 @@ spec:
     # No values or 0 means that the operator will wait until the placement groups are healthy before unblocking the next drain.
     pgHealthCheckTimeout: 0
 
+  # csiDriverOptions defines defines CSI Driver settings applied per cluster.
+  csiDriverOptions:
+    # common settings for all CSI drivers.
+    common:
+      readAffinity:
+        # set enabled to true to enable read affinity.
+        enabled: false
+        # CRUSH location labels define which node labels to use
+        # as CRUSH location. This should correspond to the values set in
+        # the CRUSH map.
+        # Defaults to all the labels mentioned in
+        # https://rook.io/docs/rook/latest/CRDs/Cluster/ceph-cluster-crd/#osd-topology
+        # crushLocationLabels:
+        # - "kubernetes.io/hostname"
+        # - "topology.kubernetes.io/zone"
+        # - "topology.rook.io/rack"
+
+    # cephfs driver specific settings.
+    cephfs:
+      # Set CephFS Kernel mount options to use https://docs.ceph.com/en/latest/man/8/mount.ceph/#options
+      # Set to "ms_mode=secure" when connections.encrypted is enabled in CephCluster CR
+      # kernelMountOptions: "ms_mode=secure"
+      # Set CephFS Fuse mount options to https://docs.ceph.com/en/quincy/man/8/ceph-fuse/#options
+      # fuseMountOptions: ""
```

Comment:
Is fuse supported now, or still just experimental with ceph csi?

Reply:
> Is fuse supported now, or still just experimental with ceph csi?

There are still downsides but it is in better shape.

Reply:
curious to discuss more another time on this topic.